c_examples/pause.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
```

Comment:
Maybe adding a configurable `sleep` time in between will make this example code more useful.